packages/suite/src/reducers/suite/suiteReducer.ts

```diff
@@ -489,12 +484,26 @@ export const selectFirmwareHashCheckError = (state: StateForFirmwareChecks) => {
     return isCheckEnabled && checkResult?.success === false ? checkResult.error : null;
 };
 
+/**
+ * Determine if there is a FW hash check error that could be caused by outdated Suite version (a subset of firmware hash error)
+ */
+export const selectIsUnrecognizedFirmwareWithOutdatedSuite = (state: AppState): boolean => {
+    const device = selectDevice(state);
+    if (!isDeviceAcquired(device) || !device.authenticityChecks?.firmwareHash) return false;
+    const isUpdateAvailable = state.desktopUpdate.state === UpdateState.Available;
```

Comment:
I verified that `desktopReducer` is in the basic set of reducers common for all apps, so it is guaranteed to be on state.
If I understand correctly, it gets updated by actions which are **only** triggered on desktop (`suite-desktop-ui`)